_maps/corgstation.json

```diff
@@ -1,5 +1,6 @@
 {
     "map_name": "Corg Station",
+	"map_link": "CorgStation",
```

Comment:
Inconsistent indentation, should be spaces

Reply:
whoops, will fix